cluster/gce/gci/configure.sh

```diff
@@ -518,11 +518,6 @@ function install-containerd-ubuntu {
   # Override to latest versions of containerd and runc
   systemctl stop containerd
   if [[ -n "${UBUNTU_INSTALL_CONTAINERD_VERSION:-}" ]]; then
-    # TODO(https://github.com/containerd/containerd/issues/2901): Remove this check once containerd has arm64 release.
-    if [[ $(dpkg --print-architecture) != "amd64" ]]; then
-      echo "Unable to automatically install containerd in non-amd64 image. Bailing out..."
-      exit 2
-    fi
```

Comment:
instead of remove the check here, i'd suggest to be conservative, as it's not clear the status of other arches, e.g. ppc64le

```if [ $(dpkg --print-architecture) != "amd64" ] && [ $(dpkg --print-architecture) != "arm64" ]; then```

Reply:
Thanks @chendave 
Emm... Just follow the comment instruction `Remove this check once containerd has arm64`.
I have no objection to your suggestion, but it seems more cleaner to just get rid of this check. 
IMO, if an unsupported arch type is provided, I think the following download process will fail and the script's behavior will not be changed.

Reply:
> I think the following download process

download might successfully, but the support in the containerd might be experiential / not fully verified.

etcd use to have arm64 supported with only experimental status for quite a long time.

Let's leave to @SergeyKanzhelev, he knows gce better than me.

Reply:
I'd just go with removing this logic. There is not much else on GCE to run this script on =)

/lgtm
/approve